site/content/en/docs/administration/basics/installation.md

```diff
@@ -47,31 +44,28 @@ For access from China, read [sources for users from China](#sources-for-users-fr
   sudo apt-get update
   sudo apt-get --no-install-recommends install -y docker-ce docker-ce-cli containerd.io
   ```
-
-- Perform [post-installation steps](https://docs.docker.com/install/linux/linux-postinstall/)
-  to run docker without root permissions.
+More instructions can be found [here](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
+3. Perform [post-installation steps](https://docs.docker.com/install/linux/linux-postinstall/) to run docker without root permissions.
```

Comment:
Need to have an empty line.

<img width="921" alt="image" src="https://user-images.githubusercontent.com/40690625/194919171-abfa9c53-78a7-4173-9042-b5e52fa186d4.png">